android/libraries/rib-base/src/main/java/com/badoo/ribs/core/plugin/Plugin.kt

```diff
@@ -4,15 +4,26 @@ import android.os.Bundle
 import android.view.ViewGroup
 import androidx.lifecycle.Lifecycle
 import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.Rib
 import com.badoo.ribs.core.view.RibView
 
 interface Plugin
 
+interface RibAware<T : Rib> : Plugin {
+    val rib: T
+
+    fun init(rib: T) {}
+}
+
 interface NodeAware : Plugin {
```

Comment:
can we pass View type to this interface, router know about view, so there will be less stars in code)

Reply:
Router doesn't know about view though. It's only for RoutingAction<View> which will change to RoutingAction very soon as it turned out it's not needed, so don't worry about that.